services/src/main/java/org/keycloak/authentication/AbstractFormAuthenticator.java

```diff
@@ -22,6 +22,7 @@
  *
  * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
  * @version $Revision: 1 $
+ * HELLO WORLD
```

Comment:
Is this intentional change? :-)

Reply:
Nopes, used it to test the CodeQL conditional jobs and forgot to remove it. Removed now.